package.json

```diff
@@ -1,36 +1,66 @@
 {
-  "name": "loopback-connector-cos",
-  "version": "0.1.0",
-  "description": "IBM Cloud Object Storage connector for LoopBack",
-  "engines": {
-    "node": ">=6"
-  },
-  "scripts": {
-    "test": "mocha",
-    "lint": "eslint --ignore-path .gitignore .",
-    "posttest": "npm run lint"
+  "_args": [
```

Comment:
Please remove all properties starting with `_`, I believe they are internal metadata added by `npm`.

Also try to preserve existing content in this file, just add new dependencies required by your code.